packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/en.ftl

```diff
@@ -1,5 +1,6 @@
-verificationReminderSecond-subject = Final reminder: Activate your account
-verificationReminderSecond-title = Still there?
-verificationReminderSecond-description = Almost a week ago you created a { -product-firefox-account } but never verified it. We’re worried about you.
+verificationReminderSecond-subject-2 = Remember to confirm your account
+verificationReminderSecond-title-2 = Don't miss out on { -brand-firefox }
+verificationReminderSecond-description-3 = A few days ago you created a { -product-firefox-account }, but never confirmed it. Please confirm your account in the next 10 days or it will be automatically deleted.
+verificationReminderSecond-second-description = Your { - product-firefox-account} lets you sync your info across devices and unlocks access to more privacy-protecting products from Mozilla.
```

Comment:
```suggestion
verificationReminderSecond-second-description = Your { - product-firefox-account } lets you sync your info across devices and unlocks access to more privacy-protecting products from { -brand-mozilla }.
```

Reply:
Will do, thank you!

Reply:
Thanks for adding the placeholder! I'm just noticing this now but this placeholder is a bit wonky for verificationReminderSecond-second-description: 
`{ - product-firefox-account}`
should be:
`{ -product-firefox-account }`

There's a space in-between the dash and the name, and a space missing before the final bracket.

Reply:
Thank you for catching that! Updating now